Source/WebCore/dom/Element.cpp

```diff
@@ -2232,7 +2238,7 @@ void Element::attributeChanged(const QualifiedName& name, const AtomString& oldV
         break;
     case AttributeNames::idAttr: {
         AtomString oldId = elementData()->idForStyleResolution();
-        AtomString newId = makeIdForStyleResolution(newValue, document().inQuirksMode());
+        AtomString newId = makeIdForStyleResolution(newValue, document->inQuirksMode());
```

Comment:
Nit: auto*?

Reply:
Sorry can you clarify which line you mean?

Reply:
I'm not sure. Maybe these `AtomString` declarations, but they should be `auto` presumably, not `auto*`.